Pod/Classes/WPMediaCollectionViewController.m

```diff
@@ -209,8 +209,16 @@ - (void)refreshTitle {
     } else {
         self.titleButton.hidden = NO;
     }
-    NSString *title = [NSString stringWithFormat:@"%@ %@", [mediaGroup name], ArrowDown];
-    [self.titleButton setTitle:title forState:UIControlStateNormal];
+    NSString *localizedOptionHint = NSLocalizedString(@"Tap here to change", "Tip for tapping media picker title to change the group.");
+    NSString *albumName = [NSString stringWithFormat:@"%@\n", [mediaGroup name]];
```

Comment:
We need to default to something like "No Photos" or an empty string if there's no media group. For example, if the app doesn't have permission to access photos, you see `(null)`:

<img width="159" alt="screen shot 2017-02-28 at 13 47 07" src="https://cloud.githubusercontent.com/assets/4780/23407634/ce89378c-fdbc-11e6-9574-277e3c56bc4a.png">